src/main/java/gyro/google/dns/PolicyResource.java

```diff
@@ -0,0 +1,276 @@
+     */
+    // XXX: https://github.com/perfectsense/gyro/issues/190
+    @Updatable
+    public DnsPolicyAlternativeNameServerConfig getAlternativeNameServerConfig() {
+        return alternativeNameServerConfig;
+    }
+
+    public void setAlternativeNameServerConfig(DnsPolicyAlternativeNameServerConfig alternativeNameServerConfig) {
+        this.alternativeNameServerConfig = alternativeNameServerConfig;
+    }
+
+    /**
+     * A mutable string of at most 1024 characters associated with this resource for the user's convenience. Has no effect on the policy's function. The value may be ``null``.
+     */
+    @Required
+    @Updatable
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    /**
+     * Allows networks bound to this policy to receive DNS queries sent by VMs or applications over VPN connections. When enabled, a virtual IP address will be allocated from each of the sub- networks that are bound to this policy. The value may be ``null``.
+     */
+    @Updatable
+    public Boolean getEnableInboundForwarding() {
+        return enableInboundForwarding;
+    }
+
+    public void setEnableInboundForwarding(Boolean enableInboundForwarding) {
+        this.enableInboundForwarding = enableInboundForwarding;
+    }
+
+    /**
+     * Controls whether logging is enabled for the networks bound to this policy. Defaults to no logging if not set. The value may be ``null``.
+     */
+    @Updatable
+    public Boolean getEnableLogging() {
+        return enableLogging;
+    }
+
+    public void setEnableLogging(Boolean enableLogging) {
+        this.enableLogging = enableLogging;
+    }
+
+    /**
+     * Unique identifier for the resource; defined by the server (output only). The value may be ``null``.
+     */
+    @Output
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    /**
+     * User assigned name for this policy. The value may be ``null``.
+     */
+    @Required
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    /**
+     * List of network names specifying networks to which this policy is applied. The value may be ``null``.
+     *
+     * @subresource gyro.google.dns.DnsPolicyNetwork
+     */
+    // XXX: https://github.com/perfectsense/gyro/issues/190
+    @Updatable
+    public List<DnsPolicyNetwork> getNetwork() {
+        return network;
+    }
+
+    public void setNetwork(List<DnsPolicyNetwork> network) {
+        this.network = network;
+    }
+
+    @Override
+    public boolean doRefresh() throws Exception {
+        Dns client = createClient(Dns.class);
+        Policy policy = client.policies().get(getProjectId(), getName()).execute();
+        return refreshFrom(policy);
```

Comment:
Following convention.
```suggestion
        
        return refreshFrom(policy);
```

Reply:
I see your point, but honestly I wouldn't be bothered if those kinda convention can't be automated by checkstyle or some tools since there's no way we can enforce them 100%, which could lead to wasting of time and slow down the reviewing process.
If you still think those conventions should be enforced, please capture it in README or in some place so that everyone can follow the rules.